rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -119,10 +119,8 @@ components:
       required: true
       schema:
         type: string
-        minLength: 32
-        maxLength: 32
-        pattern: '^[0-9a-zA-Z]+$'
-        example: 00067360b70e4acfab561fe593ad3f7a
+        format: uuid
+        example: 00067360-b70e-4acf-ab56-1fe593ad3f7a
```

Comment:
:question: Do you want to restrict the format? 

      `pattern: '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'`

Reply:
We can either use a regex or the UUID class, not both. Patrick decided to use UUID class.

Reply:
I see. I read the OpenAPI documentation and it said `Formats such as "email", "uuid", and so on, MAY be used even though undefined by this specification.` I thought it was going to treat it as a String, but I just looked at the generated code and it uses the UUID class. :+1: